### DISCUSSION TITLE
Block Sponsored Element in Followed Channels Sidebar

### DISCUSSION THREAD

**Discussion by syd-h** [night/betterttv#6741] [Ideas]
![image](https://github.com/night/betterttv/assets/25559485/b11cf8d4-1ae3-4f5a-8a8b-dafd16297d91)

A toggle to block this element in the followed channels side bar. Not sure if it's new or A/B testing but this is the first I've seen it pop up. 

It seems easy enough to block as just an element for something like uBlock Origin. But I figured it was a good request since the plugin has other minor cosmetic toggles.

This was the element that currently blocks it on my side. 
##.tw-link.side-nav-card__link--promoted-followed.side-nav-card__link

**Comment by night** [comment]
Closing this one out — can't reproduce, the sponsored element in the followed channels sidebar doesn't appear to be present anymore. Please open a new discussion if it comes back.
